index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Lisandro-Subrat</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <div id="container">
+      <div id="header">
+        <h1 id="title">survey</h1>
+        <p id="description">Thanks for filling the survey!</p>
+      </div>
+
+      <form id="survey-form">
+        <div class="form-control">
+          <label id="name-label">Name:</label>
+          <input
+            type="text"
+            placeholder="Enter your name"
+            name="name"
+            id="name"
+            required
+          />
+        </div>
+        <div class="form-control">
+          <label id="email-label">Email:</label>
+          <input
+            type="email"
+            placeholder="Enter your email"
+            name="email"
+            id="email"
+            required
+          />
+        </div>
+        <div class="form-control">
+          <label id="number-label">age:</label>
+          <input
+            type="number"
+            placeholder="Enter your age"
+            name="age"
+            id="number"
+            min="10"
+            max="100"
+            required
+          />
+        </div>
+        <div class="form-control">
+          <label>Dropdown options:</label>
+          <select name="Dropdown" id="dropdown">
+            <option disabled="" selected="" value="">Select an option</option>
+            <option value="option1">Option 1</option>
+            <option value="option2">Option 2</option>
+            <option value="option3">Option 3</option>
+            <option value="option4">Option 4</option>
+          </select>
+        </div>
+        <div class="form-control">
+          <h2>Radio Options :</h2>
+          <input type="radio" id="age1" name="age" value="30" />
+          <label>option 1</label><br />
+          <input type="radio" id="age2" name="age" value="60" />
+          <label>option 2</label><br />
+          <input type="radio" id="age3" name="age" value="100" />
+          <label>option 3</label><br /><br />
+        </div>
+        <div class="form-control">
+          <h2>Check Boxes</h2>
+          <input
+            type="checkbox"
+            id="checkbox1"
+            name="checkbox1"
+            value="checkbox1"
+          />
+          <label for="checkbox1"> checkbox 1</label><br />
+          <input
+            type="checkbox"
+            id="checkbox2"
+            name="checkbox2"
+            value="checkbox2"
+          />
+          <label for="checkbox2"> checkbox 2</label><br />
+          <input
+            type="checkbox"
+            id="vehicle3"
+            name="checkbox3"
+            value="checkbox3"
+          />
+          <label> checkbox 3</label><br /><br />
+        </div>
+
+        <div class="form-control" id="text-div">
+          <p>aditional comments:</p>
```

Comment:
- [ ] 'aditional comments' it's more a title for the input than a paragraph, so you should consider to use the <hx> tag instead the \<p> tag :face_with_spiral_eyes: :, try something like this:
```suggestion
          <hx>aditional comments:</hx>
``` 
#1

Reply:
- [ ] I think that it would be nice to explain briefly why this change should be implemented, this way project owners can learn and not repeat the same mistake again ☀️